web/src/main/java/com/futurewei/alcor/web/entity/route/RouteTable.java

```diff
@@ -29,15 +29,15 @@
     private List<RouteEntry> routeEntities;
 
     @JsonProperty("route_table_type")
-    private RouteTableType routeTableType;
+    private String routeTableType;
```

Comment:
Keep RouteTableType if possible.

Reply:
If I keep RouteTableType, it is difficult to mock requestBody in UTs